R/style_guides.R

```diff
@@ -75,7 +75,8 @@ tidyverse_style <- function(scope = "tokens",
         add_line_break_before_curly_closing,
       partial(
         set_line_break_after_opening_if_call_is_multi_line,
-        except_token = "COMMENT"
+        except_token_after = "COMMENT",
+        except_text_before = c("switch", "ifelse", "if_else")
```

Comment:
(Basically answering "yes" to your earlier question, if I understood correctly.)